sphinx/io.py

```diff
@@ -50,37 +53,6 @@ class SphinxBaseReader(standalone.Reader):
     """
     Add our source parsers
     """
-    def __init__(self, app, parsers={}, *args, **kwargs):
```

Comment:
I moved a responsibility of choosing a parser to `read_doc()` function.
In docutils, it is not a responsibility of reader. so I dropped it from here.